pkg/image/source.go

```diff
@@ -35,6 +37,7 @@ const SchemeSeparator = ":"
 
 var sourceStr = [...]string{
 	"UnknownSource",
+	"ContainerdDaemonSource",
```

Comment:
Should this just be `ContainerdDaemon` like `DockerDaemon` and `PodmanDaemon`?

Reply:
good find